madhav/src/main/java/com/lftechnology/phpjava/main/Task.java

```diff
@@ -0,0 +1,28 @@
+package main.java.com.lftechnology.phpjava.main;
+
+import main.java.com.lftechnology.phpjava.numbers.AddMultiples;
+import main.java.com.lftechnology.phpjava.numbers.Fibonacci;
+import main.java.com.lftechnology.phpjava.strings.NameSplitter;
+
+public class Task {
+    public void runTask(int taskNumber) {
+        switch (taskNumber) {
+        case 1:
+            NameSplitter ns = new NameSplitter();
+            ns.getAndExtractname();
+            break;
+        case 2:
+            AddMultiples am = new AddMultiples();
+            am.getMultiples();
+            break;
+        case 3:
+            Fibonacci f = new Fibonacci();
+            f.getFibonacciSum();
+            break;
+        case 0:
+        default:
+            System.out.println("Bye!");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS106)